src/components/ProtectedRoute.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
```

Comment:
[Optional] You might consider implementing props validation, as this is essential to catch bugs during development